### DISCUSSION TITLE
zh/games/gomoku

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#8] [Q&A]
# zh/games/gomoku

开源的 React 五子棋游戏在线版。无需下载，现在就开始你的五子棋对战！

https://gallery.selfboot.cn/zh/games/gomoku

**Comment by RealPeopleOfChina** [comment]
真想知道如何使用Claude。

**Reply by selfboot** [reply]
现在国内注册比较容易被封号。不过你可以买 cursor 会员，它有 claude3.5 模型的
